cluster/addons/dns/kube-dns.yaml.base

```diff
@@ -48,6 +48,33 @@ metadata:
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:kube-dns
```

Comment:
The "system:" prefix is reserved for resources created by the API server. Consider naming this just "kube-dns"

Reply:
done!